adam-core/src/main/scala/org/bdgenomics/adam/util/TwoBitFile.scala

```diff
@@ -92,6 +92,16 @@ class TwoBitFile(byteAccess: ByteAccess) extends ReferenceFile {
     name -> contigOffset
   }
 
+  def getSequenceDictionary(performLexSort: Boolean = false): SequenceDictionary = {
+    val sd = new SequenceDictionary(seqRecords.toVector.map(r => SequenceRecord(r._1, r._2.dnaSize)))
+
+    if (performLexSort) {
```

Comment:
IIRC, we've added lex sort functions to the `SequenceDictionary`, so I'd rather not dupe the code.